test/test_intensify/test_abstract_intensifier.py

```diff
@@ -48,6 +49,33 @@ def setUp(self):
 
         self.logger = logging.getLogger(self.__module__ + "." + self.__class__.__name__)
 
+    def test_get_next_challenger(self):
+        """
+            test get_next_challenger - pick from list/chooser
+        """
+        intensifier = AbstractRacer(
+            tae_runner=None, stats=self.stats, traj_logger=None,
+            rng=np.random.RandomState(12345), deterministic=True, run_obj_time=False,
+            cutoff=1, instances=[1], initial_budget=1, max_budget=3, eta=2)
+
+        # None when nothing to choose from
+        config = intensifier.get_next_challenger(challengers=None, chooser=None, run_history=self.rh)
```

Comment:
Hm, shouldn't this result in an exception?

Reply:
When you cannot repeat configurations, this gives a chance for exiting gracefully in the loop or try again. But I can also make it raise an error if that would be better.

Reply:
Thanks a lot for the change, but you also need to remove the comment above.